### PR TITLE
y e s

### PR DESCRIPTION
**Pull request by GramingFoxTeam**
just y e s

### CODE REVIEW

File: 2020/10/y e s.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Never gonna give you up, never gonna let you down
+Never gonna run around and desert you
+Never gonna make you cry, never gonna say goodbye
+Never gonna tell a lie and hurt you
+
+Never gonna give you up, never gonna let you down
+Never gonna run around and desert you
+Never gonna make you cry, never gonna say goodbye
+Never gonna tell a lie and hurt you
```
Comment (jessephus) [inline 2020/10/y e s.md:58]: So good. 😂 

Rick Roll

**Review by Blario123 — APPROVED** [review]

File: 2020/10/y e s.md (hunk 1 above)
Comment (samuei) [reply 2020/10/y e s.md:58]: 😲 
But that's copyrighted! Oh no!

**Review by starikcetin — APPROVED** [review]
LGTM

**Comment by sallamadim** [comment]
sanana yarram

File: 2020/10/y e s.md (hunk 1 above)
Comment (reisxd) [reply 2020/10/y e s.md:58]: ohno i will get dmcad

**Review by n0rmancodes — APPROVED** [review]

**Comment by jessephus** [comment]
Hi :wave: I am doing the inevitable and closing this pull request because we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to this repository. Feel free to keep the discussion going in this thread. Please just be mindful to keep the discussion respectful and consistent with [GitHub’s Community Guidelines](https://docs.github.com/github/site-policy/github-community-guidelines).

We understand that you may have questions about this takedown. Although we do not normally monitor comments posted to this repository, our team is reading through the comments and questions in this and other recent threads to see where we may be able to clarify our policies or answer some common questions. Thanks.

**Comment by flleeppyy** [comment]
Props to Jeese for not loosing it over what's going on right now

**Comment by n0rmancodes** [comment]
> Props to Jeese for not loosing it over what's going on right now

agreed

**Comment by GramingFoxTeam** [comment]
> Hi :wave: I am doing the inevitable and closing this pull request because we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to this repository. Feel free to keep the discussion going in this thread. Please just be mindful to keep the discussion respectful and consistent with [GitHub’s Community Guidelines](https://docs.github.com/github/site-policy/github-community-guidelines).
> 
> We understand that you may have questions about this takedown. Although we do not normally monitor comments posted to this repository, our team is reading through the comments and questions in this and other recent threads to see where we may be able to clarify our policies or answer some common questions. Thanks.
but you didnt closed others..?

**Pull request closed without merging**